README.md

```diff
@@ -15,11 +15,17 @@ find discrepancies between the coverage report and your suite's behavior, please
 + `solidity-coverage` is [Solcover](https://github.com/JoinColony/solcover)
 
 ### Install
+
+Run the following command in a Truffle based application or the root directory of your solidity project.
+
```

Comment:
In principle you *can* run `solidity-coverage` on non-Truffle projects. The only requirement is that the instrumentation targets be located in a `contracts` folder.

It's not explicit but the installation command itself contains the necessary information about how / where to install e.g. the `--save-dev` flag says: this package should be installed in the local `node_modules` as a development dependency. 

I can see how that might not be clear to people who are unfamiliar with NPM though. Maybe this line could be simplified a little? To simply state in plain english that solidity-coverage is a local dev dependency?

Reply:
@designium I wonder if you could clarify something: you're using truffle as a test runner without using truffle's project organization. Why? How might using Jenkins require a different project pattern? 

I think it would be fantastic to have a section in the [FAQ](https://github.com/sc-forks/solidity-coverage/blob/master/docs/faq.md) addressing non-truffle installations (for example -using with Embark). Would also love to see guides to other CI setups. But a compelling use case has to be developed and they need to have a meaningful working examples at their core.

Reply:
Another framework that would be nice to see an example installation on is [Dapp / Dapple](https://dapp.readthedocs.io/en/latest/).

Reply:
@cgewecke Thanks for the discussion. 

Just clarifying, I'm not using Truffle test without Truffle. I was just replying to your previous question of what test framework I'm using. 

For simple contracts, I would love to use solidity coverage that is not based on Truffle. Just simple checks. That is why I am trying to run solidity-coverage in a non-Truffle based contract.

Ideally, solidity-coverage could be run by simply:

    $solidity-coverage FILENAME.sol

or 

    $solidity-coverage /contracts/*

If that syntax works like that, without much dependencies, then it will be very easy to integrate with other testing frameworks and CI.

I understand that you are interested in including solidity-coverage in other testing frameworks.